how-to/how-to-use-rootless-kata-containers-with-podman.md

```diff
@@ -0,0 +1,275 @@
+# How to use rootless Kata Containers with Podman
+
+* [How to use rootless Kata Containers with Podman](#how-to-use-rootless-kata-containers-with-podman)
+    * [Requirements](#requirements)
+    * [Installation](#installation)
+    * [Configuration](#configuration)
+        * [Disable SELinux](#disable-selinux)
+        * [Add user to KVM group](#add-user-to-kvm-group)
+        * [Reboot](#reboot)
+        * [Disable `vhost-net`](#disable-vhost-net)
+        * [Modify the Kata images permissions](#modify-the-kata-images-permissions)
+        * [Set up Podman rootless configuration](#set-up-podman-rootless-configuration)
+        * [Add Kata Runtime to Podman configuration file (optional)](#add-kata-runtime-to-podman-configuration-file-optional)
+    * [Run Kata with rootless Podman](#run-kata-with-rootless-podman)
+    * [Appendix: Possible Errors](#appendix-possible-errors)
+        * [Error caused by agent or runtime version mismatch](#error-caused-by-agent-or-runtime-version-mismatch)
+        * [Missing registry file](#missing-registry-file)
+
+For an even more secure system, [Kata Containers](https://Katacontainers.io)
+can run workloads without a privileged user. Using
+[Podman](https://podman.io/) as the container engine, and
+[`slirp4netns`](https://github.com/rootless-containers/slirp4netns) for
+user-space networking.
+
+## Requirements
+
+- A Linux system, see
+  [supported distributions](https://github.com/kata-containers/documentation/blob/master/install/README.md#supported-distributions)
+  for an updated list.
+
+  - If using CentOS 7, `newuidmap` and `newgidmap` do not exist. Install them with:
+
+    ```bash
+    $ (git clone https://github.com/shadow-maint/shadow; cd shadow; ./autogen.sh --prefix=/usr --enable-man; make && sudo make -C src install)
+    $ sudo bash -c 'echo 10000 > /proc/sys/user/max_user_namespaces'
+    $ sudo bash -c "echo $(whoami):110000:65536 > /etc/subuid"
+    $ sudo bash -c "echo $(whoami):110000:65536 > /etc/subgid"
+    ```
+
+- Golang 1.12
+
+## Installation
+
+Refer to the following table for the __minimum__ required software versions
+and the installation instructions:
+
+| Component       | Version       | Install Instructions|
+| ----------------|:-------------:|---------------------|
+| Podman          | 1.6.2         | [see here](https://github.com/containers/libpod/blob/master/install.md)
+| `slirp4netns`   | 0.4.0         | [see here](https://github.com/rootless-containers/slirp4netns#quick-start)
+| Kata Containers | 1.10.0-alpha1 | [see here](https://github.com/kata-containers/documentation/blob/master/install/README.md)
+
+
+> **NOTE:**
+>
+> If installing Podman with a package manager, there is usually no need to
+> install `slirp4netns` separately.
+> You will need to install Kata Containers from source today for rootless support.
```

Comment:
I installed using kata-deploy using the 1.10.0-alpha1 tag so this is not technically true. You could rephrase to "You will need to install Kata Containers from source today for rootless support, or you could use [kata-deploy](https://github.com/kata-containers/packaging/tree/master/kata-deploy) with a 1.10.0-alpha1 tag."